infra/fastly/terraform/dl.k8s.io/vcl/binaries.vcl

```diff
@@ -0,0 +1,144 @@
+sub vcl_recv {
+  # configure purges to require api authentication:
+  # https://docs.fastly.com/en/guides/authenticating-api-purge-requests
+  #
+  if (req.method == "FASTLYPURGE") {
+      set req.http.Fastly-Purge-Requires-Auth = "1";
+  }
+
+  # Prevent edge from caching stale content served from shield
+  # https://developer.fastly.com/learning/concepts/stale/#shielding-considerations
+  if (fastly.ff.visits_this_service != 0) {
+    set req.max_stale_while_revalidate = 0s;
+  }
+
+
+#FASTLY recv
+
+  # don't bother doing a cache lookup for a request type that isn't cacheable
+  if (req.method !~ "(GET|HEAD|FASTLYPURGE)") {
+    return(pass);
+  }
+
+  return(lookup);
+}
+
+sub vcl_fetch {
+
+  if (beresp.status >= 200 && beresp.status < 300) {
+    set beresp.cacheable = true;
+  }
+
+  /* handle 5XX (or any other unwanted status code) */
+  if (beresp.status >= 500 && beresp.status < 600) {
+    /* deliver stale if the object is available */
+    if (stale.exists) {
+      return(deliver_stale);
+    }
+
+    if (req.restarts < 1 && (req.request == "GET" || req.request == "HEAD")) {
+      restart;
+    }
+
+    set beresp.cacheable = true;
```

Comment:
Not sure this should be true? Do you want to cache 5xx responses?

Reply:
No. we don't want to cache 5xx responses. will switch to `false`.